src/model/sort-order.ts

```diff
@@ -0,0 +1,4 @@
+export enum SortOrder {
+  Increase = 'asc',
```

Comment:
It's okay to give string values to enums and use them in other places(e.g. in API params in your case), but usually, it adds coupling to your components and it makes enums harder to update(because they are highly coupled with other functional)
To couple enum to some string(e.g. enum to query params in our case) we can introduce another mapper, what will be coupled to other functionality:
```ts
export enum SortOrder { ... }

export const SortOrderToQueryParams: Record<SortOrder, string> = {
  [SortOrder.Increase]: 'asc',
  ...
}
```

Reply:
and then use this way:
```js
const increaseParam = SortOrderToQueryParams[SortOrder.Increase]
```